js/Utils.js

```diff
@@ -133,3 +133,7 @@ function capitalize(s){
     if (typeof s !== 'string') return ''
     return s.charAt(0).toUpperCase() + s.slice(1)
 }
+
+function removeTrailingZeros(value) {
+    return (value + '').replace(/(\.\d*[1-9])0+/, '$1');
```

Comment:
```suggestion
    var lastchar = value.charAt(value.length-1);
    if (lastchar == "0") {
        return (value + '').replace(/(\.\d*[1-9])0+/, '$1');
    } else {
        return value;
    }
```
I'm trying to use this function in the amount value of tx confirmation popup (which is still WIP).
When the amount is 0.000101010101, it returns 0.00010101011. So it should run only when the last digit is 0.

Reply:
Fixed